src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingService.kt

```diff
@@ -111,6 +114,8 @@ class VilkårsvurderingService(
 
         val eksisterendeVilkårResultater = personResultat.vilkårResultater
 
+        adopsjonService.oppdaterAdopsjonsdato(behandlingId = BehandlingId(behandlingId), aktør = personResultat.aktør, nyAdopsjonsdato = endreVilkårResultatDto.adopsjonsdato, vilkår = endreVilkårResultatDto.endretVilkårResultat.vilkårType)
+
```

Comment:
Har valgt å trekke sjekken på om det er barnets alder inn i selve funksjonen, men kan også gjøre det her slik at oppdateringsfunksjonen ikke trenger å ta inn vilkår. Noen tanker?

Reply:
Oops, jeg så ikke denne kommentaren før jeg kommenterte. Men stemmer for at det bør trekkes ut! :D